includes/controllers/pages/class-submit-listing.php

```diff
@@ -890,7 +890,11 @@ private function plan_selection() {
 
             if ( ! $plan || ! $plan->enabled || ! $plan->supports_category_selection( $categories ) ) {
                 if ( $this->editing ) {
-                    $this->messages( _x( 'Please choose a valid category for your plan.', 'submit listing', 'business-directory-plugin' ), 'error', 'plan_selection' );
+					if ( ! $plan->enabled ) {
+						$this->messages( _x( 'Current active fee plan is disabled. Please select another fee plan.', 'submit listing', 'business-directory-plugin' ), 'error', 'plan_selection' );
```

Comment:
Let's use "plan" going forward instead of "fee plan". It's confusing since they don't have to have a fee.